python/pdo/contract/__init__.py

```diff
@@ -20,7 +20,15 @@
     "ContractResponse",
     "ContractRequest",
     "register_contract",
-    "add_enclave_to_contract"
+    "add_enclave_to_contract",
+    "ReplicationRequest",
+    "start_replication_service",
+    "stop_replication_service",
+    "add_replication_task",
+    "TransactionRequest",
+    "start_transaction_processing_service",
+    "stop_transacion_processing_service",
+    "add_transaction_task"
```

Comment:
normally these would be exported by the various modules (and the __all__ just refers to the modules)

Reply:
ideally i would have liked to export the whole module itself, somehow this did not work, so ended up exporting a whole bunch of methods